samples/Web/Startup.fs

```diff
@@ -86,6 +88,5 @@ type Startup() =
         if env.IsDevelopment() then app.UseDeveloperExceptionPage() |> ignore
         else app.UseHsts() |> ignore
 
-        app.UseHttpsRedirection()
-            .UseCors(fun x -> x.WithOrigins([|"https://www.todobackend.com"|]).AllowAnyHeader().AllowAnyMethod() |> ignore)
-            .UseMvc() |> ignore
\ No newline at end of file
+        app.UseCors(fun x -> x.AllowAnyOrigin().AllowAnyHeader().AllowAnyMethod() |> ignore)
```

Comment:
Right now I can't get the `todobackend.com` website to work with `WithOrigins([|"https://www.todobackend.com"|])` so since this is not the _main focus_ of the sample I'd suggest we make it more flexible by allowing any origin.

Reply:
relaxing is not the end of the world but per my quick tests it seems to work in https://github.com/jet/dotnet-templates/blob/master/equinox-web-csharp/Web/Startup.cs#L27 ?

Reply:
I am not sure I can't get it to work, I hit CORS issues if I try to use the non HTTPS endpoint.
And I hit ERR certificate if I want to use the HTTPS endpoint.

In order to get the sample to work,  I had to remove the HTTPS redirection and allow all origins.

Reply:
1) if the code needs to change, we'll chang it, but the correct/intended code absolutely should remain in a comment or an ifdef
2) dotnet dev-certs can fix HTTPS, but a comment would be friendlier...

Reply:
After offline discussion, the HTTPS problem can be fixed with dotnet dev-certs:

```
❯ dotnet dev-certs https --clean
❯ dotnet dev-certs https --trust
```

Thanks!